processing/text_processing.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+import pandas as pd
+import re
+import string
+import nltk
+from nltk.tokenize import word_tokenize
+from nltk.stem import WordNetLemmatizer 
+
+from nltk.corpus import stopwords
+nltk.download('stopwords')
+stop = stopwords.words('english')
+
+# project_dir = r'D:\GitHub\Projects\Analysis_of_Delivery_Companies_Reviews'
+# os.chdir(project_dir)
+
+def trimSpace(text) :
+    """
+    Takes string input, removes leading and trailing spaces and returns string output
+    
+    """
+    return str.strip(text)
+
+
+def removeNum(text) :
+    """
+    Takes string input, removes numbers and returns string output
+    
+    """
+    return re.sub('\d+', '',text)
+
+def removePunctuation(text) :

```

Comment:
We can merge the removeNum and removePunctuation into a single function, where we indicate if we want to run one of them or both (i.e. if a flag is True then run else not run).